audit/auditLog.json

```diff
@@ -237,143 +237,543 @@
-      "1.0.0": ["audit20241007"],
-      "1.2.0": ["audit20241206"]
+      "1.0.0": [
+        "audit20241007"
+      ],
+      "1.2.0": [
+        "audit20241206",
+        "audit20250508"
+      ]
+    },
+    "AcrossFacet": {
+      "2.0.0": [
+        "audit20250508"
+      ]
     },
     "AcrossFacetV3": {
-      "1.0.0": ["audit20241007"],
-      "1.1.0": ["audit20250106"]
+      "1.0.0": [
+        "audit20241007"
+      ],
+      "1.1.0": [
+        "audit20250106",
+        "audit20250508"
+      ]
+    },
+    "AllBridgeFacet": {
+      "2.0.0": [
+        "audit20250508"
+      ]
+    },
+    "AmarokFacetPacked": {
+      "1.0.0": [
+        "audit20250508"
+      ]
+    },
+    "AmarokFacet": {
+      "3.0.0": [
+        "audit20250508"
+      ]
+    },
+    "ArbitrumBridgeFacet": {
+      "1.0.0": [
+        "audit20250508"
+      ]
     },
     "CalldataVerificationFacet": {
-      "1.2.0": ["audit20240902"],
-      "1.3.0": ["audit20250220"]
+      "1.2.0": [
+        "audit20240902",
+        "audit20250508"
+      ],
+      "1.3.0": [
+        "audit20250220"
+      ]
+    },
+    "CBridgeFacetPacked": {
+      "1.0.3": [
+        "audit20250508"
+      ]
+    },
+    "CBridgeFacet": {
+      "1.0.0": [
+        "audit20250508"
+      ]
+    },
+    "CelerCircleBridgeFacet": {
+      "1.0.1": [
+        "audit20250508"
+      ]
+    },
+    "CelerIMFacetMutable": {
+      "2.0.0": [
+        "audit20250508"
+      ]
     },
     "ChainflipFacet": {
-      "1.0.0": ["audit20250305"]
+      "1.0.0": [
+        "audit20250305",
+        "audit20250508"
+      ]
     },
     "DeBridgeDlnFacet": {
-      "1.0.0": ["audit20241205"]
+      "1.0.0": [
+        "audit20241205",
+        "audit20250508"
+      ]
     },
     "DexManagerFacet": {
-      "1.0.2": ["audit20250413"]
+      "1.0.2": [
+        "audit20250413",
+        "audit20250508"
+      ]
     },
-    "EmergencyPauseFacet": {
-      "1.0.0": ["audit20240913"],
-      "1.0.1": ["audit20241105"]
-    },
-    "ERC20Proxy": {
-      "1.1.0": ["audit20250109_2"]
+    "DiamondCutFacet": {
+      "1.0.0": [
+        "audit20250508"
+      ]
     },
-    "Executor": {
-      "2.1.0": ["audit20250109_2"]
+    "DiamondLoupeFacet": {
+      "1.0.0": [
+        "audit20250508"
+      ]
     },
-    "FeeCollector": {
-      "1.0.1": ["audit20250109_3"]
+    "EmergencyPauseFacet": {
+      "1.0.0": [
+        "audit20240913"
+      ],
+      "1.0.1": [
+        "audit20241105",
+        "audit20250508"
+      ]
     },
     "GasZipFacet": {
-      "2.0.0": ["audit20241107"],
-      "2.0.2": ["audit20250110_1"],
-      "2.0.3": ["audit20250416"]
-    },
-    "GasZipPeriphery": {
-      "1.0.0": ["audit20241107"],
-      "1.0.1": ["audit20250110_1"]
-    },
-    "GenericErrors": {
-      "1.0.1": ["audit20250506"]
+      "2.0.0": [
+        "audit20241107"
+      ],
+      "2.0.2": [
+        "audit20250110_1"
+      ],
+      "2.0.3": [
+        "audit20250416",
+        "audit20250508"
```

Comment:
the cantina comp had GasZipFacet 2.0.2, this needs to be moved up

Reply:
It was modified as part of this audit